Sources/BraintreeAmericanExpress/BTAmericanExpressClient.swift

```diff
@@ -24,8 +24,9 @@ import BraintreeCore
     ///   `rewardsBalance` will contain information about the rewards balance and `error` will be `nil` (see exceptions in note);
     ///   if it fails, `rewardsBalance` will be `nil` and `error` will describe the failure.
     ///  - Note: If the nonce is associated with an ineligible card or a card with insufficient points, the rewardsBalance will contain this information as `errorMessage` and `errorCode`.
-    public func getRewardsBalance(forNonce nonce: String, currencyIsoCode: String, completion: @escaping (BTAmericanExpressRewardsBalance?, Error?) -> Void) {
-        let parameters = ["currencyIsoCode": currencyIsoCode, "paymentMethodNonce": nonce]
+    @objc(getRewardsBalanceForNonce:currencyIsoCode:completion:)
+    public func getRewardsBalance(for nonce: String, currencyISOCode: String, completion: @escaping (BTAmericanExpressRewardsBalance?, Error?) -> Void) {
```

Comment:
The `forNonce` variant might be preferred here since `String` is an opaque type in this method. The `nonce` parameter just happens to contain a string representation of a nonce.

Reply:
Yeah, technically, in an ideal world, we'd want this module to have merchants just pass in a `BTPaymentMethodNonce` type here instead of a `String`. That way merchants could pass in a `BTCardNonce` here and on the SDK side we would extract the nonce value out for them. Not really possible in it's current state but we should revisit when `Card` is in Swift since it doesn't currently subclass `BTPaymentMethodNonce` but could.

I can revert this for now, but going to add a TODO when we convert Card to revisit this.